File: Olimpiadi di informatica/23/ois_23/23.c
/*
 * This template is valid both in C and in C++,
 * so you can expand it with code from both languages.
 * NOTE: it is recommended to use this even if you don't
 * understand the following code.
 */

#include <stdio.h>
#include <assert.h>

// constraints
#define MAXT 10000

// input data
int T, i;
int N[MAXT];

int main() {
//  uncomment the following lines if you want to read/write from files
//  freopen("input.txt", "r", stdin);
//  freopen("output.txt", "w", stdout);

    assert(1 == scanf("%d", &T));
    for(i=0; i<T; i++) {
        assert(1 == scanf("%d", &N[i]));

        // insert your code here
    
        printf("%d ", 42); // print the result
    }
    printf("\n");
    return 0;
}

File: Olimpiadi di informatica/23/ois_23/23.pas
{ NOTE: it is recommended to use this even if you don't understand the following code. }

{ constraints }
const
    MAXT = 10000;

{ input data }
var
    T, i  : longint;
    N     : array[0..MAXT-1] of longint;

begin
{
    uncomment the following lines if you want to read/write from files
    assign(input,  'input.txt');  reset(input);
    assign(output, 'output.txt'); rewrite(output);
}

    readln(T);
    for i:=0 to T-1 do
    begin
        read(N[i]);
        
        { insert your code here }


        write(42); { print result }
        write(' ');
    end;
    writeln()
end.
